### PR TITLE
Kansas City 2016

### PR DESCRIPTION
**Pull request by twoflowers**
Adding the official dates

### CODE REVIEW

**Comment by bridgetkromhout** [comment]
This needs to wait for a signed venue contract.

**Comment by twoflowers** [comment]
Location has been added before the cutoff date, I believe.  I will update more as we begin to switch

File: site/content/events/2016-kansascity/index.html [hunk 1]
```diff
@@ -21,7 +21,7 @@
 <!-- <a href="./location">Location</a> -->
 <br><br>
 
-<!-- <a href="./propose">Propose a talk</a>! -->
+<a href="./propose">Propose a talk</a>!
```
Comment (barkerd427) [inline site/content/events/2016-kansascity/index.html]: Reviewing on my phone. Does this open the cfp?

**Comment by barkerd427** [comment]
+1

**Comment by chris-stein** [comment]
:+1: 

Hi @bridgetkromhout, have you received the signed vendor contract?  If you have, does it look okay?

**Comment by bridgetkromhout** [comment]
I don't need the contract. If you say you have one, you're good.

**Pull request merged**